osgp/shared/osgp-ws-distributionautomation/src/main/java/org/opensmartgridplatform/ws/da/config/DistributionAutomationWebServiceConfig.java

```diff
@@ -32,6 +32,8 @@ public PayloadValidatingInterceptor payloadValidatingInterceptor() {
         final Resource[] resources = new Resource[] { new ClassPathResource(COMMON_XSD_PATH),
                 new ClassPathResource(DISTRIBUTION_AUTOMATION_XSD_PATH) };
         payloadValidatingInterceptor.setSchemas(resources);
+        payloadValidatingInterceptor.setValidateRequest(false);
```

Comment:
Request validation now set to true. This means we validate all requests against the schema, which sounds a bit expensive to me in the context of this 'distributed-monolith' application OSGP. But other components also do it so aligned now.

Response validation is set to false everywhere, which makes sense to me. Otherwise we would be validating JAXB responses we generated ourselves.